src/views/RandomView.vue

```diff
@@ -1,9 +1,28 @@
+<script setup>
+import { ref } from "vue"
+  import BoosterCard from "../components/BoosterCard.vue"
+  const booster = ref(null)
+  const getNewFact = () => {
+    fetch("https://uselessfacts.jsph.pl/random.json?language=en")
+    .then(response => response.json())
+    .then(data => {
+      console.log(data)
+      return booster.value = data
+      })
+  }
+</script>
+
 <template>
   <div class="random">
     <h1>Generate Random Boosters</h1>
+    <div class="button-container">
```

Comment:
Kind of neat that you are back to writing slightly more standard HTML.  I might just note to keep your HTML as semantic as you can.